consensus/hotstuff/votecollector/interfaces.go

```diff
@@ -0,0 +1,74 @@
+package votecollector
+
+import (
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// SigType is the aggregable signature type.
+type SigType int
+
+// There are two signatures are aggregable, one is the normal staking signature,
+// the other is the threshold sig used as staking sigature.
+const (
+	SigTypeStaking SigType = iota
+	SigTypeThreshold
+	SigTypeInvalid
+)
+
+// CombinedAggregator aggregates signatures from different signers for the same message.
+// The instance must be initialized with the message, the required weight to be
+// sufficient for aggregation, as well as the weight for each signer.
+type CombinedAggregator interface {
```

Comment:
I defined [two](https://github.com/onflow/flow-go/pull/1089/files#diff-7d6c161422be7302d7cdcebd5ccd2d1f542542d0758ea0c439e53e7a62d1af5bR9) aggregators [separately](https://github.com/onflow/flow-go/pull/1089/files#diff-141d5952e3a8d22db4137f55e412b44b3206befd28bc3643cb48a3d565c01bdaR9) in my signing interface PR. I  think would be more clear, and since we are not going to reuse them in collection cluster.

Reply:
Yes, but the main reason for merging them into combined aggregator is `weight` which can be encapsulated inside of `CombinedAggregator`. I have thought that we have aggreged on that. Thoughts?